### ISSUE TITLE
[NeoForge][1.21.1]Traits don't pass on to combined materials.

### ISSUE THREAD

**Issue by Voided24** [SilentChaos512/Silent-Gear#736]
## Versions

- **Silent Gear**: 4.0.8
- **Silent's Gems**: x.x.x 
- **Silent Lib**: 10.3.0
- **NeoForge**: 21.1.73
- **Modpack**: ATM 10 - 1.14
- **Optifine Installed**: No 

## Expected Behaviour
Traits pass onto combined materials

## Actual Behaviour
After combining materials they lose all traits

## Links/Images
https://i.imgur.com/smf2p7E.png
https://i.imgur.com/eUAvKkx.png

## Steps to Reproduce the Problem
1. Get any 2 materials
2. Combine them in any of the combining blocks

**Comment by Voided24** [comment]
Expanding on this further as I just discovered it.
Starcharging alloyed metals yields 0 stat change whilst grading still does

**Comment by 1-peeed** [comment]
Yo, sorry but how did you get NeoForge 21.1.73?

**Comment by johskar** [comment]
> Yo, sorry but how did you get NeoForge 21.1.73?

It is the latest release for Minecraft 1.21.1 from what I can tell, and also what All the Mods 10 currently comes with.

**Comment by zero1crazy** [comment]
Is there any progress on this bug?

**Comment by SilentChaos512** [comment]
No, I haven't gotten around to it yet. The traits system has quite a few bugs in 1.21 that have been difficult to work out.

**Comment by SilentChaos512** [comment]
Closed by 42746ba1940e760e7c0e25c26a0650b84d9521a5
